### PR TITLE
Fixes issues with opset number in a couple of models

### PR DESCRIPTION
**Pull request by xadupre**

### CODE REVIEW

File: skl2onnx/operator_converters/gaussian_process.py [hunk 1]
```diff
@@ -40,6 +40,8 @@ def convert_gaussian_process_regressor(scope, operator, container):
     out = operator.outputs
     op = operator.raw_operator
     opv = container.target_opset
+    if opv is None:
+        raise RuntimeError("container.target_opset must be None")
```
Comment (prabhat00155) [inline skl2onnx/operator_converters/gaussian_process.py]: Did you want to say "container.target_opset must not be None"?

File: tests/test_op10.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+from sklearn.ensemble import VotingClassifier
+from sklearn.gaussian_process import GaussianProcessRegressor
+from sklearn.mixture import GaussianMixture, BayesianGaussianMixture
+from skl2onnx import convert_sklearn
+from skl2onnx.common.data_types import FloatTensorType
+from skl2onnx.common.data_types import onnx_built_with_ml
+
+
+class TestOp10(unittest.TestCase):
+    def _fit_model_binary_classification(self, model):
```
Comment (prabhat00155) [inline tests/test_op10.py]: These functions are defined in test_utils, you can just use that.
https://github.com/onnx/sklearn-onnx/blob/679bbdc0465ed86235e71a982b3e62edcbfb0e3d/tests/test_utils/tests_helper.py#L51

File: tests/test_op10.py [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+        iris = datasets.load_iris()
+        X = iris.data[:, :3]
+        y = iris.target
+        y[y == 2] = 1
+        model.fit(X, y)
+        return model, X
+
+    def _fit_model_multiclass_classification(self, model):
```
Comment (prabhat00155) [inline tests/test_op10.py]: https://github.com/onnx/sklearn-onnx/blob/679bbdc0465ed86235e71a982b3e62edcbfb0e3d/tests/test_utils/tests_helper.py#L51

File: tests/test_op10.py (hunk 2 above)
Comment (xadupre) [reply tests/test_op10.py]: fixed

File: tests/test_op10.py (hunk 3 above)
Comment (xadupre) [reply tests/test_op10.py]: fixed

File: skl2onnx/operator_converters/gaussian_process.py (hunk 1 above)
Comment (xadupre) [reply skl2onnx/operator_converters/gaussian_process.py]: fixed

**Pull request merged**